views/includes/value-display.pug

```diff
@@ -22,6 +22,10 @@ if (currencyValue > 0)
 		else
 			small.ml-1 #{parts.currencyUnit}#{currencyUnitSuffix}
 
+	else if (currencyFormatInfo.type == "exchanged" && currencyFormatInfo.isExtendedRate)
+		small.h5  🛈
```

Comment:
here you have a trailing char that is not supposed to be there or at least I don't see the reason why to have it. 

it also renders weirdly on the web app, see:
<img width="310" alt="Screenshot 2021-10-20 at 11 17 03" src="https://user-images.githubusercontent.com/1469203/138065303-5aef345e-98ff-4645-b4b0-257aa1bf7e7a.png">

and also
<img width="140" alt="Screenshot 2021-10-20 at 11 17 40" src="https://user-images.githubusercontent.com/1469203/138065432-4dd34c35-ed14-46ae-b430-c6628f078609.png">
 
I think we should remove `small.h5` tag entirely